_episodes/40.md

```diff
@@ -0,0 +1,481 @@
+---
+layout: episode
+title: "40: Trino's cold as Iceberg!"
+date: 2022-09-08
+tags: iceberg apache datalake table format sql schema evolution partition optimize
+youtube_id: "6NyfCV8Me0M"
+wistia_id: "b9nlw7rjpr"
+sections: 
```

Comment:
releases section is missing

Reply:
maybe also needs separate one for Trino Summit announcement?

Reply:
and an intro one??

Reply:
Not for the Intro, I assume if they want to see that they can just watch from the beginning. I forgot to announce the Trino Summit in the actual show though :(